source/architecture.rst

```diff
@@ -89,7 +89,7 @@ API Programming Example
 
 API programming requires the programmer to specify the target device and the
 memory communication strategy.  In the following example, we call the
-oneMKL matrix multiply routine, GEMM.  We are writing in SYCL and
+oneMath matrix multiply routine, GEMM.  We are writing in SYCL and
```

Comment:
So we agreed to call it the oneMath Specification and Interfaces library, but what is the full name ?  previously it was the `oneAPI Math Kernel Library (oneMath)`,  now is it `oneAPI Math (oneMath)` ?

Reply:
or is still the `oneAPI Math Kernel Library (oneMath)`?

Reply:
No, I did not realize we needed a full name and short name for the RFC. My first assumption was that we would always refer to the project as simply "oneMath".
Since we found a few occurrences of "oneAPI Math Kernel Library (oneMKL)" I think it makes sense to replace them with "oneAPI Math Library (oneMath)".